specification/_global/get/GetResponse.ts

```diff
@@ -17,29 +17,8 @@
  * under the License.
  */
 
-import { Dictionary } from '@spec_utils/Dictionary'
-import { UserDefinedValue } from '@spec_utils/UserDefinedValue'
-import {
-  Id,
-  IndexName,
-  SequenceNumber,
-  Type,
-  VersionNumber
-} from '@_types/common'
-import { long } from '@_types/Numeric'
+import { GetResult } from '@global/get/types'
 
-export class Response<TDocument> {
-  body: {
-    _index: IndexName
-    fields?: Dictionary<string, UserDefinedValue>
-    found: boolean
-    _id: Id
-    _primary_term?: long
-    _routing?: string
-    _seq_no?: SequenceNumber
-    _source?: TDocument
-    /** @deprecated 7.0.0 */
-    _type?: Type
-    _version?: VersionNumber
-  }
+export class Response<TDocument> extends GetResult<TDocument> {
```

Comment:
There is no compiler check yet, but we should not extend Response classes, as if you define a property in the extending class it will override the one defined in the extended class.

Reply:
I don't understand _"if you define a property in the extending class it will override the one defined in the extended class"_. Can you show a simple example?

From a typed language perspective, we really need to be able to use a single type to represent the result of a `get` and the hits of an `mget`. So the idea here is to use `GetResult` as the base class of the `get` response and use it directly in `mget` hits.

Reply:
I've replaced the `extends` with a value body in 011e936 as this is really what we want here: be able to use the same structure that is used for `get` responses as `mget` response items. WDYT?